spec/queries/boards/get_own_last_boards_in_history_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Boards::GetOwnLastBoardsInHistory do
+  let_it_be(:user1) { create(:user) }
+  let_it_be(:user2) { create(:user) }
+  let_it_be(:board1) { create(:board) }
+  let_it_be(:second_board1) { create(:board, previous_board_id: board1.id) }
+  let_it_be(:board2) { create(:board) }
+  let_it_be(:permission) { create(:permission, identifier: 'destroy_board') }
+
+  before do
+    create(:board_permissions_user, permission: permission,
+                                    user: user1, board: second_board1)
+  end
+
+  context 'call method' do
+    it 'returns array of boards that user\'s a creator' do
```

Comment:
We can use double quotes in order not to use a `\`, because the description is created for people. Or write `user is a creator` :D